### PR TITLE
 Closes #1716: Add overrides for claro node form layout styles.

### LINKED ISSUE #1716: Restore layout of node creation pages to use the full width of the window
## Problem/Motivation
I noticed a change was introduced to the Quickstart repo in the past few weeks which alters the layout of the node creation pages at `/node/add/<content type>`. The new layout sets a maximum width on the main form fields and centers them, such that many users will now see a lot of white space on either side. The new layout also shrinks the width of the secondary region (containing the menu settings, summary fields, etc.).

### Describe the issue
As a result of code merged between 6/10 and 6/24, the HTML of the node creation pages has changed. Nested inside the `<div class="layout-region layout-region--node-main">` element, there is now a `<div class="layout-region__content">` container around the form field elements. This container has a max-width set along with auto left and right margins. The CSS styles applied to the layout-region--node-main element have changed also. The secondary sidebar region (with the menu settings, summary fields, etc.) also includes the layout-region__content container as a child and the sidebar is now more narrow than before.

### Additional context
@trackleft noted that this issue stems from an update to the Claro theme, as discussed in [this related Drupal issue](https://www.drupal.org/project/drupal/issues/3184667).

### To Reproduce

1. Go to the Add Content page at `/node/add`
2. Click on any of the content types
3. Observe the increased margins around the main form and the reduced width of the right sidebar

## Proposed resolution
To be determined

### Expected behavior

The main form fields should make better use of the available width and the sidebar width should not be reduced, given that many site builders may use it to add a node to a menu, enter summary text for the node, and so on.

##  Screenshots

Before:
image

After:
image

### PR DESCRIPTION
**Pull request by joeparsons**
## Description
Adds CSS styles that override some Claro styles that were added in 9.4.x that make the content editing UI layout not ideal, especially for Paragraphs content.

## Related issues

Closes #1716 

## How to test

Edit content, inspect UI

### Arizona Quickstart (install profile, custom modules, custom theme)
- Patch release changes
   - [x] Bug fix
   - [ ] Accessibility, performance, or security improvement
   - [ ] Critical institutional link or brand change
- Minor release changes
   - [ ] New feature
   - [ ] Breaking or visual change to existing behavior
   - [ ] Non-critical brand change
   - [ ] New internal API or API improvement with backwards compatibility
   - [ ] Risky or disruptive cleanup to comply with coding standards
   - [ ] High-risk or disruptive change (requires upgrade path, risks regression, etc.)
- [ ] Other or unknown

### Drupal core
- Patch release changes
   - [ ] Security update
   - [ ] Patch level release (non-security bug-fix release)
   - [ ] Patch removal that's no longer necessary
- Minor release changes
   - [ ] Major or minor level update
- [ ] Other or unknown

### Drupal contrib projects
- Patch release changes
   - [ ] Security update
   - [ ] Patch or minor level update
   - [ ] Add new module
   - [ ] Patch removal that's no longer necessary
- Minor release changes
   - [ ] Major level update
- [ ] Other or unknown

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: modules/custom/az_core/az_core.module [hunk 1]
```diff
@@ -240,3 +241,10 @@ function az_core_form_block_form_alter(&$form, &$form_state, $form_id) {
     $form['settings']['block_branding']['#description'] = t('Choose which branding elements you want to show in this block instance.  Enabling both the Site logo and the Site name is <strong><em>not recommended</em></strong> per University of Arizona brand guidelines.');
   }
 }
+
+/**
+ * Implements hook_form_BASE_FORM_ID_alter() for \Drupal\node\NodeForm.
```
Comment (joeparsons) [inline modules/custom/az_core/az_core.module:246]:
```suggestion
 * Implements hook_form_BASE_FORM_ID_alter() for \Drupal\node\NodeForm.
 *
 * Adds style overrides for claro/node-form.
```

**Review by bberndt-uaz — DISMISSED** [review]

File: modules/custom/az_core/css/claro-node-form-overrides.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
+@media screen and (min-width: 61rem) {
+  .layout-region--node-main .layout-region__content, .layout-region--node-footer .layout-region__content {
+    max-width: 100% !important;
+    margin-right: auto !important;
```
Comment (joshuasosa) [inline modules/custom/az_core/css/claro-node-form-overrides.css:4]: These margin right and left properties appear to be in Claro already.

File: modules/custom/az_core/css/claro-node-form-overrides.css [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    margin-left: auto !important;
+  }
+}
+
+@media screen and (min-width: 61rem) {
+  .layout-region--node-main, .layout-region--node-footer {
```
[the remark this answers is not in the document]
Comment (joeparsons) [reply modules/custom/az_core/css/claro-node-form-overrides.css]: Thanks for reviewing @joshuasosa!  I've removed the redundant styles.

**Review by trackleft — APPROVED** [review]

**Comment by trackleft** [comment]
Claro node-form https://git.drupalcode.org/project/drupal/-/blob/9.5.x/core/themes/claro/claro.libraries.yml
Claro CSS https://git.drupalcode.org/project/drupal/-/blob/9.5.x/core/themes/claro/css/layout/node-add.css

**Review by bberndt-uaz — APPROVED** [review]

**Review by tadean — APPROVED** [review]

**Pull request merged**